### ISSUE TITLE
Set-Up brings up a redundant menu 

### ISSUE THREAD

**Issue by chrisfosterelli** [stevelyall/project-ernie-app#28]
Model: Nexus 5

The menu that appears when you select "Set-Up" has two options: Consent & Back.

There isn't anything else on the screen, and the consent screen itself has a back button. The actual "Set-Up" screen itself appears pretty redundant from the user's perspective, and could just be skipped through.

![image](https://cloud.githubusercontent.com/assets/2072672/6991462/3d85e99c-da4b-11e4-87f8-006b43aa22b3.png)

**Comment by stevelyall** [comment]
I agree- the original spec for the app had the user going into this menu to agree to the consent, provide demographics and set locations among other things- I feel like it might be simpler just to guide them through it the first time through the app?

I agree that this is redundant.

**Comment by chrisfosterelli** [comment]
I like your suggestion. I did find it somewhat confusing that there was a bit of initial setup, then a setup menu. It took some playing around to get my bearings with it. I think just making it part of the initial run makes the most sense.
